prow/plugins/stage/stage.go

```diff
@@ -0,0 +1,120 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package stage defines a Prow plugin that defines the stage of
+// the issue in the features process. Eg: alpha, beta, stable.
+package stage
```

Comment:
Please add this package to https://github.com/kubernetes/test-infra/blob/master/prow/hook/plugins.go

Reply:
I added it in https://github.com/kubernetes/test-infra/pull/8340 because it says `// All enabled plugins.` at the top of this file. :thinking:

Reply:
updated. ptal.

Reply:
Feel free to improve that comment while you're in there 😁

This more means plugins enabled at the prow level, which are "enable-able" by various repos.

Any plugin which isn't in this list won't exist in the hook binary, and so when you add a ` - stage` line in `plugins.yaml` for your repo or org then hook will look for a package that registered a "stage" plugin, fail to find one and refuse to apply the update.